test_plans/spark-processing

```diff
@@ -0,0 +1,21 @@
+bundle: bundle:spark-processing
+benchmark:
+    spark:
+        logisticregression
+    spark:
```

Comment:
Can we add multiple benchmarks? A lot of solutions include more than one benchmark action